src/providers/SocketProvider.tsx

```diff
@@ -58,6 +58,13 @@ export const SocketProvider: React.FunctionComponent = ({ children }: any) => {
       setUniqueID(payload['uniqueID']);
     } else if (msgType === MESSAGE_TYPES.QUEUE_LIST) {
       setQueue(payload['queueMembers']);
+    } else if(msgType === MESSAGE_TYPES.JOIN_CHAT){
+      console.log("Noen ville legge meg til i chatten!");
+      console.log(payload['roomID']);
+      const student:IStudent = {nickname: "", course: "", grade: "", uniqueID: "", introText:""};
```

Comment:
```suggestion
      const student:IStudent = {nickname: '', course: '', grade: '', uniqueID: '', introText:''};
```

Linteren er ganske streng, så før du pusher ut en PR som ikke er WIP så bør du sette opp webstorm sånn at den formatterer slik linteren (prettier med typescript) vil, hvis ikke kommer jenkins til å klage